snow/engine/snowman/block/state_syncable_vm.go

```diff
@@ -6,9 +6,16 @@ package block
 import (
 	"context"
 	"errors"
+
+	"github.com/ava-labs/avalanchego/ids"
 )
 
-var ErrStateSyncableVMNotImplemented = errors.New("vm does not implement StateSyncableVM interface")
+var (
+	ErrStateSyncableVMNotImplemented = errors.New("vm does not implement StateSyncableVM interface")
```

Comment:
This is the main change of this PR, where the new interface is introduced and its usage described